Dockerfile

```diff
@@ -101,6 +101,49 @@ RUN  cd /tmp/api-gateway/ \
     && rm /usr/local/test-nginx-${TEST_NGINX_VERSION}.tar.gz \
     && ln -s /usr/local/sbin/api-gateway-debug /usr/local/sbin/nginx
 
+ENV CONFIG_SUPERVISOR_VERSION initial-poc
+ENV GOPATH /usr/lib/go/bin
+ENV GOBIN  /usr/lib/go/bin
+ENV PATH   $PATH:/usr/lib/go/bin
+RUN echo " ... adding api-gateway-config-supervisor  ... " \
+    && echo "http://dl-4.alpinelinux.org/alpine/edge/community" >> /etc/apk/repositories \
+    && apk update \
+    && apk add make git go \
```

Comment:
is this the same container as the one being deployed? If so, perhaps different containers should be used for build and runtime (see http://www.hokstad.com/docker/patterns)